Thorfile

```diff
@@ -0,0 +1,6 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
```

Comment:
I was curious to know if RuboCop recognizes `Thorfile` as Ruby and it does! Nice.